plugin/src/test/java/org/opensearch/ml/utils/TestHelper.java

```diff
@@ -138,4 +146,32 @@ public static String httpEntityToString(HttpEntity entity) throws IOException {
         }
         return sb.toString();
     }
+
+    public static DataFrame constructTestDataFrame(int size) {
```

Comment:
nitpick: should this be defined in the TestData class since it's creating data for tests?

Reply:
Will address in next PR.